crates/iota-rest-kv/src/extractors.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module includes custom extractors needed for validation and custom
+//! errors messages on the input provided by the client
+
+use core::str;
+
+use axum::{
+    async_trait,
+    extract::{FromRequestParts, Path as AxumPath},
+    http::request::Parts,
+};
+use iota_storage::http_key_value_store::{Key, TaggedKey};
+use iota_types::{
+    digests::{CheckpointDigest, TransactionDigest},
+    storage::ObjectKey,
+};
+use serde::Deserialize;
+
+use crate::{errors::ApiError, types::ItemType};
+
+/// Path segment labels will be matched with struct field names
+#[derive(Deserialize, Debug)]
+struct DigestAndTypeParams {
+    /// The digest encoded as [base64_url]
```

Comment:
Does this `[base64_url]` actually link to anything?

Reply:
@DaughterOfMars yes, if I press it, this web page appears: https://docs.rs/base64-url/2.0.2/base64_url/index.html

Reply:
@DaughterOfMars [fixed](https://github.com/iotaledger/iota/pull/5296/commits/b32b1c994797957a8aff57fa371ba240ebe0ef85)